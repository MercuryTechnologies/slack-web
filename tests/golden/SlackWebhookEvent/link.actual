EventEventCallback
    ( EventCallback
        { eventId = EventId
            { unEventId = "Ev043R67DQ1H" }
        , teamId = TeamId
            { unTeamId = "T043DB835ML" }
        , eventTime = MkSystemTime
            { systemSeconds = 1663978925
            , systemNanoseconds = 0
            }
        , event = EventMessage
            ( MessageEvent
                { blocks =
                    [ RichText
                        { blockId = Just
                            ( NonEmptyText "MEdM" )
                        , elements =
                            [ RichTextSectionItemRichText
                                [ RichItemLink
                                    ( RichLinkAttrs
                                        { style = RichStyle
                                            { rsBold = False
                                            , rsItalic = False
                                            }
                                        , url = "https://jadeapptesting.slack.com/archives/C043YJGBY49/p1663961604007869"
                                        , text = Nothing
                                        }
                                    )
                                ]
                            ]
                        }
                    ]
                , channel = ConversationId
                    { unConversationId = "C043YJGBY49" }
                , text = "<https://jadeapptesting.slack.com/archives/C043YJGBY49/p1663961604007869>"
                , channelType = Channel
                , user = UserId
                    { unUserId = "U043H11ES4V" }
                , ts = "1663978925.099999"
                , threadTs = Nothing
                , appId = Nothing
                , botId = Nothing
                }
            )
        }
    )