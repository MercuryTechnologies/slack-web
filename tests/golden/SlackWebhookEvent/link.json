{
  "token": "aaaa",
  "team_id": "T043DB835ML",
  "api_app_id": "A0442TUPHGR",
  "event": {
    "client_msg_id": "ade2a33a-3aa8-4fcf-9b93-2e44e4a92899",
    "type": "message",
    "text": "<https://jadeapptesting.slack.com/archives/C043YJGBY49/p1663961604007869>",
    "user": "U043H11ES4V",
    "ts": "1663978925.099999",
    "team": "T043DB835ML",
    "blocks": [
      {
        "type": "rich_text",
        "block_id": "MEdM",
        "elements": [
          {
            "type": "rich_text_section",
            "elements": [
              {
                "type": "link",
                "url": "https://jadeapptesting.slack.com/archives/C043YJGBY49/p1663961604007869"
              }
            ]
          }
        ]
      }
    ],
    "channel": "C043YJGBY49",
    "event_ts": "1663978925.099999",
    "channel_type": "channel"
  },
  "type": "event_callback",
  "event_id": "Ev043R67DQ1H",
  "event_time": 1663978925,
  "authorizations": [
    {
      "enterprise_id": null,
      "team_id": "T043DB835ML",
      "user_id": "U0442US8QGH",
      "is_bot": true,
      "is_enterprise_install": false
    }
  ],
  "is_ext_shared_channel": false,
  "event_context": "aaaa"
}
